index.bs

```diff
@@ -828,12 +870,14 @@ partial interface HTMLIFrameElement {
                   parser</a> on |element|.
                 1. If |result| is not failure:
                     1. Let |target| be the origin of |result|.
-                    1. If |target| is not an opaque origin, append |target| to
+                    1. If |target| is not an opaque origin, append the tuple |target|, <code>False</code> to
                       |allowlist|.
         1. Set |directive|[|feature|] to |allowlist|.
     1. Return |directive|
 
     </div>
+    
+    Note: This algorithm does not support wildcard subdomains in allowlists.
```

Comment:
Is it deliberate that wildcards are not supported in the attribute syntax? (Just that if not -- if it's something to fix -- it should be an Issue rather than a note)

Reply:
It is deliberate, I added another sentence to that effect